fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -2236,4 +2236,11 @@ public class Config extends ConfigBase {
 
     @ConfField
     public static boolean enable_execute_script_on_frontend = true;
+
+
+    /**
+     * The default scheduler interval for alter jobs and routine jobs.
+     */
+    @ConfField(mutable = true)
+    public static int default_scheduler_interval_millisecond = 10000;
```

Comment:
```suggestion
    public static int scheduler_interval_seconds = 10;
```
is a better name. The scheduler interval is not less than one second.

Reply:
To speedup UT, it may be set to a very small value?

Reply:
Yeah, some unit tests overwrite the previous value to 1 millisecond.